core/src/main/java/com/sequenceiq/cloudbreak/controller/ExceptionControllerAdvice.java

```diff
@@ -123,4 +124,11 @@
                         e.getValue())),
                 HttpStatus.CONFLICT);
     }
+
+    @ExceptionHandler({ CredentialVerificationException.class })
+    public ResponseEntity<ExceptionResult> credentialVerification(CredentialVerificationException e) {
+        MDCBuilder.buildMdcContext();
+        LOGGER.error(e.getMessage(), e);
+        return new ResponseEntity<>(new ExceptionResult(e.getMessage()), HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
Maybe it would be better to return the errorDetails (cause message) as well to the user because it may contain a more specific error rather than "Failed to verify the credential"